sources/packages/load-test/src/_tests_/workflow/workflow-assessment-submission.test.ts

```diff
@@ -14,7 +14,7 @@ import { ClientSecretCredential } from "../../utils/auth";
 /**
  * Load test number of iterations to run.
  */
-const ITERATIONS = 500;
+const ITERATIONS = 15;
```

Comment:
Is this change on purpose to keep the number low for now?

Reply:
It was not intentional 😊